### DISCUSSION TITLE
FXIOS-1455 ⁃ Apple Store Upload issue on Xcode 12.3

### DISCUSSION THREAD

**Discussion by ChiY119** [mozilla-mobile/firefox-ios#7983] [General]
When app is uploading to Apple Store, I got "error itms-90035"...
MozillaAppServices.framework/sdk_generator.sh is not preperly signed.

I got this error message. 
Thanks

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXIOS-1455)

**Comment by dnarcese** [comment]
@ChiY119 Hello, is there a reason you are trying to upload the app to the app store? Have you forked the codebase into your own app?
